implementations/rust/ockam/ockam_api/src/authenticator/direct.rs

```diff
@@ -98,7 +98,9 @@ where
                 ["members"] => match self.check_enroller(&req, from).await {
                     Ok(None) => {
                         let add: AddMember = dec.decode()?;
-                        let tru = minicbor::to_vec(true)?;
+                        let member_attributes =
+                            HashMap::from([(ROLE.to_string(), MEMBER.to_string())]);
```

Comment:
Will all member attribute values be just strings?

Reply:
good question! :) 
I think in the near future it would be.   But we need to have a more broad conversation about this,   as the underling data struct is likely what would end up on credentials,  and what the  ABAC implementation run on top of.

Another thing to come back latter is the whole "schema update" of things stored on the storage.  The approach here is a workaround,  but would be better if we have a way to update _all_  legacy values on the storage to the new schema.   What complicates that right now is that there are different modules possible putting things on this storage, and we would like to update the schema for one of them, but not touch the values of the other ones.